diff --git a/node_modules/@types/babel__traverse/index.d.ts b/node_modules/@types/babel__traverse/index.d.ts
index 7c8be47..4553737 100644
--- a/node_modules/@types/babel__traverse/index.d.ts
+++ b/node_modules/@types/babel__traverse/index.d.ts
@@ -841,10 +841,24 @@ export class NodePath<T = Node> {
     assertRegexLiteral(opts?: object): void;
 }
 
-export class Hub {
+type ErrorConstructor = typeof Error
+
+export interface HubInterface {
+    getCode(): string | undefined;
+    getScope(): Scope | undefined;
+    addHelper(name: string): Object;
+    buildError(node: Object, msg: string, Error: ErrorConstructor): Error;
+  }
+
+
+export class Hub implements HubInterface {
     constructor(file: any, options: any);
     file: any;
     options: any;
+    getCode(): string | undefined;
+    getScope(): Scope | undefined;
+    addHelper(name: string): Object;
+    buildError(node: Object, msg: string, Error: ErrorConstructor): Error;
 }
 
 export interface TraversalContext {
